istioctl/pkg/multicluster/remote_secret.go

```diff
@@ -486,6 +486,7 @@ func applyYAML(client kube.CLIClient, yamlContent, ns string) error {
 	if err := client.ApplyYAMLFiles(ns, yamlFile); err != nil {
 		return fmt.Errorf("failed applying manifest %s: %v", yamlFile, err)
 	}
+	os.Remove(yamlFile)
```

Comment:
There is another place with the same function name `applyYAML` that has this issue. I thought I had made a similar fix before, but I don't remember why I didn't submit the PR.

Reply:
thanks for review, The other  Already `applyYAML` has added